README.md

```diff
@@ -85,22 +91,54 @@ In order to change the display configuration, you can use the polyfill together
 
 #### Manually changing the display configuration
 
-The configuration object is available as
-
-```js
-  const config = window["__foldables_env_vars__"];
-```
-
 You can update values such as `spanning`, `foldSize` and `browserShellSize` by calling the `update` method:
 
 ```js
-  config.update({
+  update({
     spanning: "single-fold-horizontal",
     foldSize: 30,
     browserShellSize: 20
   });
 ```
 
+#### Special note on web components and lit-element
+In order for the polyfill to work with web components and lit-element, a bit of extra work is needed. There are a couple of examples in the `demo/` directory.
+
+The polyfill provides two methods methods, one to register and one to observe and adjust to updates.
```

Comment:
Method methods ... Nope :) one is enough

Reply:
Fixed